e2e/specs/mattermost/back_button.test.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+'use strict';
+
+const fs = require('fs');
+
+const env = require('../../modules/environment');
+const {asyncSleep} = require('../../modules/utils');
+
+describe('Menu/window_menu', function desc() {
+    this.timeout(30000);
+
+    const config = {
+        ...env.demoConfig,
+        teams: [
+            {
+                name: 'community',
+                url: 'https://community.mattermost.com',
+                order: 0,
+                tabs: [
+                    {
+                        name: 'TAB_MESSAGING',
+                        order: 0,
+                        isOpen: true,
+                    },
+                    {
+                        name: 'TAB_FOCALBOARD',
+                        order: 1,
+                        isOpen: true,
+                    },
+                    {
+                        name: 'TAB_PLAYBOOKS',
+                        order: 2,
+                        isOpen: true,
+                    },
+                ],
+                lastActiveTab: 0,
+            },
+        ],
+    };
+
+    beforeEach(async () => {
+        env.cleanDataDir();
+        env.createTestUserDataDir();
+        env.cleanTestConfig();
+        fs.writeFileSync(env.configFilePath, JSON.stringify(config));
+        await asyncSleep(1000);
+        this.app = await env.getApp();
+        this.serverMap = await env.getServerMap(this.app);
+    });
+
+    afterEach(async () => {
+        if (this.app) {
+            await this.app.close();
+        }
+    });
+
+    it('MM-T2633 Back button should behave as expected', async () => {
```

Comment:
Nit: this title is a bit vague, can we clarify it a bit more?

Reply:
It kinda covers a large number of cases, which is why it's purposely vague.